src/Stan/Inspection/AntiPattern.hs

```diff
@@ -98,10 +109,20 @@ stan0203 = mkAntiPatternInspection (Id "STAN-0203") "Data.ByteString.Char8.pack"
         , nameMetaName       = "pack"
         }
 
-lenPatAst :: PatternAst
-lenPatAst = range
-    (PatternAstConstant 0)
-    (app
-        (PatternAstName (mkBaseFoldableMeta "length") (?))
-        (?)
-    )
+-- TODO: also warn on 'length'
```

Comment:
There's already a separate issue to support such a way of defining inspections: https://github.com/kowainik/stan/issues/87